console.log((function(n) {
    try {
        return (null.p = n = 1);
    } catch (r) {
        return n ? "PASS" : "FAIL";
    }
})());
